src/NuGet.Clients/PackageManagement.UI/Resources.resx

```diff
@@ -442,7 +442,8 @@ Packages installed into Website projects will not be restored during build. Cons
     <value>by {0}</value>
   </data>
   <data name="Text_Downloads" xml:space="preserve">
-    <value>{0} downloads</value>
+    <value> downloads</value>
```

Comment:
I would be paranoid about the space getting lost during loc, but I can see that this would be a pain to change in the xaml.